generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -547,16 +567,18 @@ Scenario: Running a 'matchingRegex' request as part of a non-contradicting allOf
        | "c"  |
        | "d"  |
 
-Scenario: Running a 'matchingRegex' request as part of a contradicting allOf constraint should fail with an error message
+  @ignore #may need to raise issue
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501